docs/precommit.md

```diff
@@ -29,29 +29,21 @@ Install it along with [husky](https://github.com/typicode/husky):
 <!--npm-->
 
 ```bash
-npm install --save-dev pretty-quick husky
+npx husky-init
+npm install --save-dev pretty-quick
+npx husky set .husky/pre-commit "pretty-quick --staged"
```

Comment:
We don't need install `husky`?

Reply:
I think it is now common practice for people to use `npx` to run CLI tools such as `husky` or `create-react-app` without having to install first. It looks to me like the first step is to use `husky-init` exactly once, then do setup to get Husky working with `pretty-quick`. I must admit I don't personally understand this whole solution so well.

Reply:
`husky-init` is one time thing, it don't need install, but `husky` runs every time you commit (at least I think it work this way before v5)

Reply:
Oh, `husky-init` [installs `husky`](https://github.com/typicode/husky/blob/a66271d21fdf798b291d009c213ed48737dae227/packages/husky-init/src/index.ts#L20) make sense.